app/src/main/java/com/teamagam/gimelgimel/app/map/actions/timeplay/TimeplayViewModel.java

```diff
@@ -200,4 +252,61 @@ public void clearMap() {
       mDisplayed.clear();
     }
   }
+
+  public class TextTimeDisplayer implements RadialTimePickerDialogFragment.OnTimeSetListener {
```

Comment:
I'd name it listener and not displayer.
This is because it's a listener and not a displayer.

Reply:
ok